pkg/gencmd/app.go

```diff
@@ -0,0 +1,8 @@
+package gencmd
+
+import "github.com/izumin5210/grapi/pkg/cli"
+
+type App struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type App should have comment or be unexported